hopsworks-api/src/main/java/io/hops/hopsworks/api/util/UploadService.java

```diff
@@ -292,23 +346,19 @@ public Response testMethod(@Context HttpServletRequest request, @Context Securit
           }
           udfso.touchz(new Path(this.path, fileName));
         } catch (AccessControlException ex) {
-          throw new AccessControlException(
-              "Permission denied: You can not upload to this folder. ");
+          throw new AccessControlException("Permission denied: You can not upload to this folder. ");
         } finally {
           dfs.closeDfsClient(udfso);
         }
       }
     }
     ResumableInfo info = getResumableInfo(request, this.path, this.templateId);
-    if (info.isUploaded(new ResumableInfo.ResumableChunkNumber(
-        resumableChunkNumber))) {
+    if (info.isUploaded(new ResumableInfo.ResumableChunkNumber(resumableChunkNumber))) {
       json.setSuccessMessage("Uploaded");//This Chunk has been Uploaded.
-      return noCacheResponse.getNoCacheResponseBuilder(Response.Status.OK).
-          entity(json).build();
+      return noCacheResponse.getNoCacheResponseBuilder(Response.Status.OK).entity(json).build();
     } else {
       json.setErrorMsg("Not uploaded");
-      return noCacheResponse.getNoCacheResponseBuilder(
-          Response.Status.NO_CONTENT).entity(json).build();
+      return noCacheResponse.getNoCacheResponseBuilder(Response.Status.NO_CONTENT).entity(json).build();
```

Comment:
see comments above - I think we are getting rid of `noCacheResponse`. Also refers to other methods in this class

Reply:
Not using it on new end-points, this one is not new